dashboard/app/assets/javascripts/toArray.filter.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+  'use strict';
+
+  angular.module('qat-dashboard').filter('toArray', function() {
```

Comment:
@mDibyo instead of copy / pasting this in our own repo, could we use the npm package instead?
https://www.npmjs.com/package/angular-toarrayfilter